direct/tpm2/get_random_test.go

```diff
@@ -0,0 +1,24 @@
+package tpm2
```

Comment:
(not a blocking comment) how do we want to organize tests? having a test per command may make the directory quite large. we could have a test directory like in `tpm2` or we could split by use case.

Reply:
My mild opinion is that I like having more and smaller test files. A test directory with e.g., one file per command tested would be groovy.

Reply:
I generally agree this makes tests easier to find, and we should go for this. I do wonder about tests that use multiple commands like a potential policy auth test with multiple policies.

Reply:
Right, and we'd have a lot of helpers too. So we might have some per-command tests as well as some "integration-y" tests that really cover a bunch of commands. That's a good reason not to be rigid on "one file per command" but in general I think it's ok to have a lot of files.